open_optical_gating/app/app.py

```diff
@@ -275,14 +281,18 @@ def emulate():
 def run():
     """Live run page."""
     # Global variable for analyse_camera object
-    # Note: this means the developer has to manage the temporal nature of accesssin this!
+    # Note: this means the developer has to manage the temporal nature of accessing this!
+    # TODO: JT writes: what on earth do the above two lines of comments mean?
     global analyse_camera
     logger.debug('analyse_camera object: {0}',analyse_camera)
 
     # TODO need to be able to set the stage z bounds for the glasgow set up, e.g. state=bounds?
+    # TODO: JT writes: so the "state" can be False, "get", "set" or "run"? That seems a bit weird to me. Are these states defined by the flask(?) API, or by you?
```

Comment:
OK, got it. I wasn't sure, since "get"/"set" etc are also HTTP concepts. And the "set" makes more sense now I understand better how manual the web interface is.
I still think the "False" is a bit incongruous as a forth option, and that merits a comment (or a string, unless there's a compelling API reason why False has to be suported.

Reply:
OK, got it. I wasn't sure, since "get"/"set" etc are also HTTP concepts. And the "set" makes more sense now I understand better how manual the web interface is.
I still think the "False" is a bit incongruous as a forth option, and that merits a comment (or a string, unless there's a compelling API reason why False has to be suported.